packages/embark/src/lib/modules/contracts_manager/index.js

```diff
@@ -287,7 +286,7 @@ class ContractsManager {
           self.contractsFiles &&
           self.contractsFiles.every(contractFile =>
             Object.values(self.compiledContracts).find(contract =>
-              contract.originalFilename === contractFile.filename
+              contract.originalFilename === path.normalize(contractFile.originalPath)
```

Comment:
That's fine, but I guess the better solution would be to have all paths normalized by default.
However, I know that some paths need to have forward slashes, so maybe it will cause issues there. To investigate,

Reply:
I may be wrong, but I think `core/file` and the paths it tracks can correspond to URLs of `http` contracts.

I agree it's not an ideal fix but I was concerned normalizing all the paths might lead to additional bugs so went for the low hanging fruit with this PR.